### ISSUE TITLE
Accessibility/Usability: Skip to main content not translated for Spanish version

### ISSUE THREAD

**Issue by iamjolly** [usagov/vote-gov#438]
## Issue:
The skip to main content link is presented in English for the Spanish version of the page.

## Screenshot:
![Screen Shot of skip to main content link in English on the Spanish version of the page](https://user-images.githubusercontent.com/1093423/112519005-8f44ea80-8d5f-11eb-95b9-7177cb9f2c42.png)

## Recommended fix:
Provide the skip link text in the language of the page. This can likely be done dynamically using new parameters set in the [config.toml](https://github.com/usagov/vote-gov/blob/staging/config.toml) file.

If you cannot translate the link quickly, and interim fix to meet Section 508 requirements would be to add the `lang="en"` attribute to the link until the translated link text is provided.

**Comment by sinchagrenchik** [comment]
Thank you for notifying us.

We will fix it.

Sincha Grenchik

USAGov - “Your Guide to Government Information and Services”

GSA, Technology Transformation Service, TTS Solutions

202-834-6389 (Cell)

**Comment by sinchagrenchik** [comment]
It should be fixed.

**Comment by iamjolly** [comment]
Thanks @sinchagrenchik! Looks good to me now. :)
